pages/Juice.md

```diff
@@ -0,0 +1,5 @@
+Juice is a [Food](Food.md) item added by Slimefun.
```

Comment:
This pages should rather be called "Juices" since it represents multiple types of Juices, not just one Juice.